src/index.html

```diff
@@ -12,7 +12,43 @@
       href="styles/index.scss"
     />
```

Comment:
The href attribute in the link tag points to a '.scss' file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Consider linking to the compiled CSS file instead.